samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -222,7 +222,7 @@ void main(void)
 	/* query IMEI */
 	query_modem("AT+CGSN", imei_buf, sizeof(imei_buf));
 	/* use IMEI as unique endpoint name */
-	snprintf(endpoint_name, sizeof(endpoint_name), "nrf-%s", imei_buf);
+	snprintf(endpoint_name, sizeof(endpoint_name), "%s%s", CONFIG_NRFCLOUD_CLIENT_ID_PREFIX, imei_buf);
```

Comment:
`lwm2m_client` shall not use this config.

Reply:
Ok, I think we can ignore lwm2m_client for this PR.